test/livebook_teams/web/hub/edit_live_test.exs

```diff
@@ -252,5 +252,28 @@ defmodule LivebookWeb.Integration.Hub.EditLiveTest do
              |> element("span", "Default")
              |> has_element?()
     end
+
+    test "use the default hub as default for new notebooks", %{hub: hub} do
+      Hubs.set_default_hub(hub.id)
+      notebook = Livebook.Session.default_notebook()
+
+      assert notebook.hub_id == hub.id
+    end
+
+    test "fallback to personal-hub when there's no default", %{hub: hub} do
+      Hubs.unset_default_hub(hub.id)
+      notebook = Livebook.Session.default_notebook()
+
+      assert notebook.hub_id == "personal-hub"
+    end
```

Comment:
Let's move these to `notebook_test.exs`.

Also, we can't run these tests asynchronously, because concurrent tests would use the default hub unexpectedly. I would put them in a separate module, but still inside `NotebookTests`, we can define a nested `defmodule Global` with `use ExUnit.Case, async: false`.

Reply:
^ `session_test.exs` in that case :)

Reply:
I think I did that correctly 😄